content/en/apps/guides/privacy/privacy-policy.md

```diff
@@ -31,6 +31,8 @@ There are two ways to add or edit a privacy policy:
 ```bash
 cht --local upload-privacy-policies
 ```
+Ensure the privacy-policies.json file is placed in the root of the project directory.
```

Comment:
We might not need here. The `cht upload` command above will show the warning if it can't find the json file:
```
INFO Actions:
     - upload-privacy-policies
INFO Starting action: upload-privacy-policies…
WARN No policies language mapping file found at path: /path/to/config/master/privacy-policies.json
INFO upload-privacy-policies complete.
```

If the user notices only the last line `INFO upload-privacy-policies complete.`, it can still get confusing.